test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,100 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps starting from a local directory containing the git repo as follows:
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc.
+
+    ```bash
+    cd test/scale-infra
+    ansible-playbook deploy_vpc_core.yaml -e "region=us-east-2 vpc_name=<vpc_name>"
+    ansible-playbook create-ec2.yaml -e "cluster_name=core region=us-east-2 vpc_name=<vpc_name> aws_key_name=mykey  num_masters=1 num_workers=2 instance_type=t2.xlarge arch=x86_64 ec2_image='<your-aws-ami>'"
```

Comment:
Testing commit c26424ad703851ccd646234fb7525fd8e7e1b0b9 just now, I still get a warning about a variable named "query".

```console
mspreitz@mjs13 scale-infra % ansible-playbook create-ec2.yaml -e "cluster_name=core region=us-east-2 vpc_name=mspreitz aws_key_name=mjs12-ed25519  num_masters=1 num_workers=2 instance_type=t2.xlarge arch=x86_64 ec2_image='ami-09d4fd64c23f3a750'"
...
TASK [ec2instances : Create ec2 instance(s)] **********************************************************************
changed: [localhost]
[WARNING]: Found variable using reserved name: query
...
```

Reply:
This is now fixed. 

```console
TASK [ec2instances : Create ec2 instance(s)] ***********************************************************************************************
changed: [localhost]

TASK [ec2instances : Get private IPs] ******************************************************************************************************
ok: [localhost]

TASK [ec2instances : Get public IPs] *******************************************************************************************************
ok: [localhost]

TASK [ec2instances : include_tasks] ********************************************************************************************************
skipping: [localhost]


``